package.json

```diff
@@ -1,50 +1,38 @@
 {
-  "name": "typewiz",
```

Comment:
it's recommended to add engines entry with minimal yarn version. yarn do validate it
When workspaces involved, i would go for at least `1.3.2`

Reply:
good point, I will add